tags/README.md

```diff
@@ -37,6 +37,6 @@ No modules.
 
```

Comment:
In description, please incldue:

This module provisions multiple DigitalOcean tags. It excludes any pre-existing tags, in order to avoid duplicate tags.